config/samples/csi-baremetal_v1_deployment.yaml

```diff
@@ -8,54 +8,57 @@ metadata:
     app.kubernetes.io/managed-by: csi-baremetal-operator
     app.kubernetes.io/version: 1.0.0
 spec:
+  globalRegistry: ""
+  nodeSelectors:
+    #key: value
   driver:
     controller:
       image:
-        name: csi-baremetal-driver
-        pullPolicy: Always
-        tag: green
+        name: csi-baremetal-controller
+        pullPolicy: IfNotPresent
+        tag: 0.0.13-375.3c20841
```

Comment:
Please leave `green` tag and `Always` pullPolicy by default.

Reply:
reset